internal/grpc/proto/kong/admin/model/v1/certificate.proto

```diff
@@ -37,4 +37,9 @@ message Certificate {
     (referenceable) = true
   ];
   repeated string tags = 8;
+  string subject = 9;
+  string issuer = 10;
+  string san = 11;
+  string expiry = 12;
+  string key_usage = 13;
```

Comment:
We should also log out the extension key usages:
https://pkg.go.dev/crypto/x509#ExtKeyUsage

We can keep it all under the same field, I see no problem combining.